piksi_multi_cpp/include/piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/ros_receiver_state.h

```diff
@@ -70,11 +71,10 @@ class RosReceiverState
         receiver_state_.fix_mode = ReceiverState::STR_FIX_MODE_DGNSS;
         break;
       case FixModeValues::kFloatRtk:
-        receiver_state_.rtk_mode_fix = 0;
         receiver_state_.fix_mode = ReceiverState::STR_FIX_MODE_FLOAT_RTK;
         break;
       case FixModeValues::kFixedRtk:
-        receiver_state_.rtk_mode_fix = 1;
+        receiver_state_.rtk_mode_fix = true;
```

Comment:
Differentiate between different GPS modes is done elsewhere. Questionable if this is not redundant

Reply:
The flag is not strictly necessary, since there is also the string, telling you which mode you're in, but it's kind of legacy thus we keep it for downwards compatibility.